src/gsi/read_prepbufr.f90

```diff
@@ -2012,17 +2012,17 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
 
 !             Over-ride QM=9 and hard-wire errors for land obs and hofx_sfc_file option
 !             Can be deleted once prepbufr processing updated.
-              if ( global_2m_land ) then
+              if ( global_2m_land .and. ignore_2mQM ) then
                 if (tob .and. qm==9 ) then
-                     pqm(k)=2 ! otherwise, type 183 will be discarded.
                      qm=2
                      tqm(k)=2
                      if (kx==187) obserr(3,k)=2.0_r_double
                      if (kx==181) obserr(3,k)=2.0_r_double
                      if (kx==183) obserr(3,k)=2.0_r_double
```

Comment:
```suggestion

```
Do you think it is necessary to discard the obs type 183 and keep consistent with new obs error table? If yes, please make change for the qob case below.

Reply:
Good point. I removed 183 from the global_2m_land option higher up.